.github/styles/Vocab/Docs/accept.txt

```diff
@@ -85,7 +85,9 @@ Nagios
 namespace
 namespaced
 (nginx|NGINX)
+NVIDIA
 Nvidia
+nvidia
```

Comment:
this makes me think we need to align on a spelling of nvidia.

Reply:
I think `nvidia` is all lowercase for some commands that we pass in and whatnot @vacas5 - I was actually just copying what we had in develop.

Reply:
I'll leave it as is for now - and then circle back on the develop branch.